helm/dashboards/dashboards/shared/public/dns.json

```diff
@@ -940,6 +940,35 @@
         "tagsQuery": "",
         "type": "query",
         "useTags": false
+      },
+      {
+        "allValue": null,
+        "current": {
+          "text": "All",
+          "value": "$__all"
+        },
+        "datasource": "$datasource",
+        "definition": "label_values(kubernetes_build_info{cluster_type=~\"workload_cluster\", cluster_id=\"$cluster\"}, organization)",
```

Comment:
This file has an 'organization` variable, but the graphs don't filter by organization; the variable seems unused.

Reply:
Thanks, I forgot to update the queries